### PR TITLE
Payara server 4.1.1.162 fix constant pool info

### PR DESCRIPTION
**Pull request by coiouhkc**
Portions Copyright [2016] [C2B2 Consulting Limited and/or its affiliates]

### CODE REVIEW

File: appserver/pom.xml [hunk 1]
```diff
@@ -98,7 +98,7 @@
         <major_version>4</major_version>
         <minor_version>1</minor_version>
         <update_version>1</update_version>
-        <payara_version>162-SNAPSHOT</payara_version>
```
Comment (lprimak) [inline appserver/pom.xml]: This change should be removed

File: appserver/pom.xml [hunk 2]
```diff
@@ -98,7 +98,7 @@
...
+        <payara_version>162</payara_version>
```
Comment (lprimak) [inline appserver/pom.xml]: This change should be removed

File: appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/annotation/introspection/ConstantPoolInfo.java [hunk 3]
```diff
@@ -122,6 +122,17 @@ public boolean containsAnnotation(int constantPoolSize, final ByteBuffer buffer)
                     buffer.getShort();
                     buffer.getShort();
                     break;
+                case METHODHANDLE:
```
Comment (lprimak) [inline appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/annotation/introspection/ConstantPoolInfo.java]: Copyright strings should be added:
// Portions Copyright [2016] [C2B2 Consulting Limited and/or its affiliates]

**Comment by lprimak** [comment]
Above should be fixed & conflicts resolved

File: appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/annotation/introspection/ConstantPoolInfo.java (hunk 3 above)
Comment (coiouhkc) [inline appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/annotation/introspection/ConstantPoolInfo.java]: to every line or just once per file at the beginning of the change?

**Comment by coiouhkc** [comment]
Regarding conflict resolution - which should be the target branch? Is master OK?

**Comment by smillidge** [comment]
thanks for the pull request. Master is fine.
For the comment it goes on the line below the Oracle copyright header

**Comment by smillidge** [comment]
We also need a CLA on file see https://github.com/payara/Payara/blob/master/Contributing.md 

Thanks for the contribution.

**Comment by coiouhkc** [comment]
Regarding the CLA - the code is to be found on stackoverflow.com, it's not my own (originating from http://stackoverflow.com/users/5026853/koen-serneels) - which implies cc by-sa 3.0 with attribution required. Is it still necessary?

**Comment by smillidge** [comment]
OK thanks we'll review and reimplement

File: appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/annotation/introspection/ConstantPoolInfo.java (hunk 3 above)
Comment (OndrejM) [inline appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/annotation/introspection/ConstantPoolInfo.java]: It is enough to add the C2B2 Copyright line once per file after the license header line in the beginning of the file. For an examples, see https://github.com/payara/Payara/blob/master/appserver/pom.xml#L43

**Pull request closed without merging**